PagerDuty Integration Information (#14648)
DataDog/documentation: content/en/tracing/faq/service_catalog.md

Click the Settings icon on the right hand corner to hide columns from the service list.

> ## Reliability view

Reviewer: this heading already exists at line 49
```suggestion
```

> #PagerDuty Integration

Reviewer: This should be an h4 if it's a sub section of h3 "Reliability view"
```suggestion
#### PagerDuty Integration
```

> You can add PagerDuty metadata to Service Catalog to complete the Reliability view. 

Reviewer: ```suggestion
You can add PagerDuty metadata to the Service Catalog to complete the Reliability view. 
```

> - Set up PagerDuty integration by following the instructions [here][6].

Reviewer: ```suggestion
- Set up the PagerDuty integration by following the instructions [on the PagerDuty integration page][6].
```

> - Get your [API acccess key][7].

Reviewer: ```suggestion
- Get your [API acccess key from PagerDuty][7].
```

> - Link PagerDuty service to Service Definition YAML.

Reviewer: ```suggestion
- Link your PagerDuty service to Service Definition YAML.
```
